lib/src/style/interactive_features/standard_poi.dart

```diff
@@ -0,0 +1,83 @@
+part of mapbox_maps_flutter;
```

Comment:
In Android/iOS we started with manual implementation of those classes, but quickly realized that there's no way to align on it, if we don't generate them from the style.

Can we generate them as well in Flutter?

Reply:
Yes: I agree on generation and will ticket